### ISSUE TITLE
Incorrect update alert in Admin panel

### ISSUE THREAD

**Issue by Marekjdj** [CTFd/CTFd#1678]
**Environment**:

- CTFd Version/Commit: 3.1.1
- Operating System: Ubuntu 20.4
- Web Browser and Version: Chrome 85

**What happened?**
The admin panel shows an alert: "A new CTFd version is available!", which links to "https://github.com/CTFd/CTFd/releases/tag/2.4.2". I encountered the issue with version 3.0.2. as well. After complete reinstall and upgrade to version 3.1.1 the problem persisted

**What did you expect to happen?**
I expected no alert, as my CTFd version is the newest, and certainly newer than 2.4.2.

**How to reproduce your issue**
Go to the admin pages.

**Any associated stack traces or error logs**
No

**Comment by ColdHeat** [comment]
Are update checks enabled? This can happen if you disable updates checking but a prior update check left data there. I think you can clear it out by setting `version_latest` in the Configs table to null.

**Comment by Marekjdj** [comment]
I never changed the updates checking and according to /CTFd/config.ini line 161 it defaults to true, so I can't see how that could be the issue.

**Comment by ColdHeat** [comment]
If that's the case then if you restart the CTFd server the update should go away after it's startup update check.

**Comment by Marekjdj** [comment]
I've tried that multiple times but it doesn't seem to work. I've also tried changing update_check(Force=False) to true (CTFd/utils/updates/__init__.py, line 15) and changing CTFd/config.py line 214 to True but it just doesn't seem to update for some reason.

**Comment by ColdHeat** [comment]
Very odd. Perhaps you can remove some of the exception handling code in https://github.com/CTFd/CTFd/blob/master/CTFd/utils/updates/__init__.py#L38-L75 to see if maybe you're getting an invalid response or not getting a response fast enough. Aside from that I don't have any idea until I can spend some time to dig into this. 

If you just want to clear out the alert I think you can just do `python manage.py set_config version_latest ""`. Normally you would set it to be None but I don't think think manage.py understands raw Python values.

**Comment by Marekjdj** [comment]
I've looked into it but can't get it to work. I guess I'm just gonna have to live with it. Thanks for your time anyway!

**Comment by ColdHeat** [comment]
I've seen this before myself so I'll re-open this issue to track progress on investigating it.

**Comment by ColdHeat** [comment]
I took a little time to look at this and the only reason I could think this could happen is if the request to the update server never actually completed. I found that if I:

1. Fake the version to a lower version
2. Let the update check go through which will show a new version in the Admin Panel
3. Then set the timeout to a very low non-zero value (e.g. 0.0000001) and restart the server
4. Then the update check request never completes and you get a requests exception which gets caught and you never get to the version checking code. 

I think this is overall the right behavior but the timeout is too low.
